components/kyma-environment-broker/internal/storage/ext.go

```diff
@@ -40,6 +40,12 @@ type Deprovisioning interface {
 	UpdateDeprovisioningOperation(operation internal.DeprovisioningOperation) (*internal.DeprovisioningOperation, error)
 }
 
+type Orchestration interface {
+	InsertOrchestration(orchestration internal.Orchestration) error
+	UpdateOrchestration(orchestration internal.Orchestration) error
+	GetOrchestrationByID(orchestrationID string) (*internal.Orchestration, error)
+}
```

Comment:
Method naming for accessing database is not unified in this case, but for me it's more elegant way:

```suggestion
type Orchestrations interface {
	Insert(orchestration internal.Orchestration) error
	Update(orchestration internal.Orchestration) error
	GetByID(orchestrationID string) (*internal.Orchestration, error)
}
```

As method signature already tells you are you inserting/updating/getting

Reply:
I can use a plural form here but unfortunately, Orchestration must be used in the function naming otherwise, it could conflict with some other interface specified in the BrokerStorage interface